### ISSUE TITLE
Updated sf cli to , getting error when running sf commands: Error Plugin: @salesforce/cli on MacOs in VS Code Terminal

### ISSUE THREAD

**Issue by TNT-c** [forcedotcom/cli#2964]
### Summary
Unable to run sf commands. Trying to view SF Debug Logs locally on VS Code terminal.
Error: `Warning: Could not find typescript. Please ensure that typescript is a devDependency. Falling back to compiled source.`

### Steps To Reproduce
Run command:` sf force:log:tail —color | grep USER_DEBUG`

### Screenshot
![image](https://github.com/user-attachments/assets/1c3b9133-3a1e-4a18-b898-36ca5b8a7e8a)

### System Info
Apple M1 Pro
14.5 (23F79)

### doctor results
```
pass - salesforcedx plugin isn’t installed
pass - you don't have any linked plugins
pass - [@salesforce/plugin-deploy-retrieve] sourceApiVersion matches apiVersion
warn - [@salesforce/plugin-deploy-retrieve] sourceApiVersion matches default target org max apiVersion
pass - can access: https://appexchange.salesforce.com/services/data
pass - can access: https://test.salesforce.com
fail - [@salesforce/plugin-auth] CLI supports v2 crypto
pass - can access: https://developer.salesforce.com/media/salesforce-cli/sf/channels/stable/sf-win32-x64-buildmanifest
pass - using latest or latest-rc CLI version
```

**Comment by mshanemc** [comment]
That's definitely an invalid command.  Did you mean `sf apex tail log --color`

**Comment by mshanemc** [comment]
how did you install that CLI?  It looks like you're trying to load dev dependencies, etc.

**Comment by TNT-c** [comment]
> sf apex tail log --color

Also getting error with the provided command
![image](https://github.com/user-attachments/assets/59af4792-4cff-4b97-af3c-0d42f5d30f72)

**Comment by TNT-c** [comment]
> how did you install that CLI? It looks like you're trying to load dev dependencies, etc.

Installed through the .pkg file
https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm#sfdx_setup_install_cli_macos

**Comment by mshanemc** [comment]
Any chance you also have the CLI installed somewhere else?  

does `npm list -g --depth 0` show sf?
does your project have sf as a dependency in its package.json?

Also, the issue instructions had a section about running `sf version --verbose` that you didn't follow.  Those are helpful for figuring out what's wrong with your setup.

**Comment by TNT-c** [comment]
> Any chance you also have the CLI installed somewhere else?
> 
> does `npm list -g --depth 0` show sf? does your project have sf as a dependency in its package.json?

Hi, This is what your provided npm command prints out:
![image](https://github.com/user-attachments/assets/0e3f2ad1-9c98-473b-a0ee-11747703c2d8)

And my package.json
```
{
  "name": "salesforce-app",
  "private": true,
  "version": "1.0.0",
  "description": "Salesforce App",
  "scripts": {
    "lint": "npm run lint:lwc && npm run lint:aura",
    "lint:aura": "eslint **/aura/**",
    "lint:lwc": "eslint **/lwc/**",
    "test": "npm run test:unit",
    "test:unit": "sfdx-lwc-jest",
    "test:unit:watch": "sfdx-lwc-jest --watch",
    "test:unit:debug": "sfdx-lwc-jest --debug",
    "test:unit:coverage": "sfdx-lwc-jest --coverage",
    "prettier": "prettier --write \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
    "prettier:verify": "prettier --list-different \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\""
  },
  "devDependencies": {
    "@prettier/plugin-xml": "^2.2.0",
    "@salesforce/eslint-config-lwc": "3.4.0",
    "@salesforce/eslint-plugin-aura": "2.1.0",
    "@salesforce/sfdx-lwc-jest": "1.4.1",
    "eslint": "8.43.0",
    "eslint-config-prettier": "8.8.0",
    "husky": "8.0.3",
    "lint-staged": "13.2.2",
    "pretier": "0.0.1",
    "prettier": "^2.8.8",
    "prettier-plugin-apex": "1.13.0"
  },
  "husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  },
  "lint-staged": {
    "**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}": [
      "prettier --write"
    ],
    "**/{aura|lwc}/**": [
      "eslint"
    ]
  }
}

```

**Comment by mdonnalley** [comment]
Do you have the `NODE_ENV` set in your environment? If so, try unsetting it to see if that resolves the issue

**Comment by TNT-c** [comment]
> Do you have the `NODE_ENV` set in your environment? If so, try unsetting it to see if that resolves the issue

How can I search for this setting? Sorry I'm not familiar. I've tried searching NODE_ENV in my project and VS Code Settings and did not find a result.

**Comment by mdonnalley** [comment]
In your terminal you can run `printenv | grep NODE_ENV`. To unset it, you can run `unset NODE_ENV`

Could you also provide the output of `DEBUG=sf:oclif:* SF_LOG_LEVEL=debug sf version`? Thanks!

**Comment by TNT-c** [comment]
> In your terminal you can run `printenv | grep NODE_ENV`. To unset it, you can run `unset NODE_ENV`
> 
> Could you also provide the output of `DEBUG=sf:oclif:* SF_LOG_LEVEL=debug sf version`? Thanks!

It appears unsetting may have resolved it. Could you ELI5 on what had happened? Thanks!
![image](https://github.com/user-attachments/assets/942ac6b2-3d64-4c36-899e-811da30a801d)


Also, here is the output that you requested:

```

  strategy: 'pattern',
  target: './lib/commands'
}
[14:34:34.094] DEBUG (sf:oclif:read-pjson): found oclif config in /usr/local/lib/sf/node_modules/@salesforce/plugin-schema/package.json
[14:34:34.094] DEBUG (sf:oclif:config:@salesforce/plugin-schema): command discovery options {
  globPatterns: [
    '**/*.+(js|cjs|mjs|ts|tsx|mts|cts)',
    '!**/*.+(d.ts|test.ts|test.js|spec.ts|spec.js|d.mts|d.cts)?(x)'
  ],
  strategy: 'pattern',
  target: './lib/commands'
}
[14:34:34.094] DEBUG (sf:oclif:read-pjson): found oclif config in /usr/local/lib/sf/node_modules/@salesforce/plugin-settings/package.json
[14:34:34.094] DEBUG (sf:oclif:config:@salesforce/plugin-settings): command discovery options {
  globPatterns: [
    '**/*.+(js|cjs|mjs|ts|tsx|mts|cts)',
    '!**/*.+(d.ts|test.ts|test.js|spec.ts|spec.js|d.mts|d.cts)?(x)'
  ],
  strategy: 'pattern',
  target: './lib/commands'
}
[14:34:34.094] DEBUG (sf:oclif:read-pjson): found oclif config in /usr/local/lib/sf/node_modules/@salesforce/plugin-sobject/package.json
[14:34:34.094] DEBUG (sf:oclif:config:@salesforce/plugin-sobject): command discovery options {
  globPatterns: [
    '**/*.+(js|cjs|mjs|ts|tsx|mts|cts)',
    '!**/*.+(d.ts|test.ts|test.js|spec.ts|spec.js|d.mts|d.cts)?(x)'
  ],
  strategy: 'pattern',
  target: './lib/commands'
}
[14:34:34.094] DEBUG (sf:oclif:read-pjson): found oclif config in /usr/local/lib/sf/node_modules/@salesforce/plugin-source/package.json
[14:34:34.094] DEBUG (sf:oclif:config:@salesforce/plugin-source): command discovery options {
  globPatterns: [
    '**/*.+(js|cjs|mjs|ts|tsx|mts|cts)',
    '!**/*.+(d.ts|test.ts|test.js|spec.ts|spec.js|d.mts|d.cts)?(x)'
  ],
  strategy: 'pattern',
  target: './lib/commands'
}
[14:34:34.094] DEBUG (sf:oclif:read-pjson): found oclif config in /usr/local/lib/sf/node_modules/@salesforce/plugin-telemetry/package.json
[14:34:34.095] DEBUG (sf:oclif:config:@salesforce/plugin-telemetry): command discovery options {
  globPatterns: [
    '**/*.+(js|cjs|mjs|ts|tsx|mts|cts)',
    '!**/*.+(d.ts|test.ts|test.js|spec.ts|spec.js|d.mts|d.cts)?(x)'
  ],
  strategy: 'pattern',
  target: './lib/commands'
}
[14:34:34.095] DEBUG (sf:oclif:read-pjson): found oclif config in /usr/local/lib/sf/node_modules/@salesforce/plugin-templates/package.json
[14:34:34.095] DEBUG (sf:oclif:config:@salesforce/plugin-templates): command discovery options {
  globPatterns: [
    '**/*.+(js|cjs|mjs|ts|tsx|mts|cts)',
    '!**/*.+(d.ts|test.ts|test.js|spec.ts|spec.js|d.mts|d.cts)?(x)'
  ],
  strategy: 'pattern',
  target: './lib/commands'
}
[14:34:34.095] DEBUG (sf:oclif:read-pjson): found oclif config in /usr/local/lib/sf/node_modules/@salesforce/plugin-user/package.json
[14:34:34.095] DEBUG (sf:oclif:config:@salesforce/plugin-user): command discovery options {
  globPatterns: [
    '**/*.+(js|cjs|mjs|ts|tsx|mts|cts)',
    '!**/*.+(d.ts|test.ts|test.js|spec.ts|spec.js|d.mts|d.cts)?(x)'
  ],
  strategy: 'pattern',
  target: './lib/commands'
}
[14:34:34.095] DEBUG (sf:oclif:read-pjson): found oclif config in /usr/local/lib/sf/node_modules/@salesforce/plugin-trust/package.json
[14:34:34.095] DEBUG (sf:oclif:config:@salesforce/plugin-trust): command discovery options {
  globPatterns: [
    '**/*.+(js|cjs|mjs|ts|tsx|mts|cts)',
    '!**/*.+(d.ts|test.ts|test.js|spec.ts|spec.js|d.mts|d.cts)?(x)'
  ],
  strategy: 'pattern',
  target: './lib/commands'
}
[14:34:34.095] DEBUG (sf:oclif:read-pjson): found oclif config in /usr/local/lib/sf/node_modules/@salesforce/plugin-limits/package.json
[14:34:34.095] DEBUG (sf:oclif:config:@salesforce/plugin-limits): command discovery options {
  globPatterns: [
    '**/*.+(js|cjs|mjs|ts|tsx|mts|cts)',
    '!**/*.+(d.ts|test.ts|test.js|spec.ts|spec.js|d.mts|d.cts)?(x)'
  ],
  strategy: 'pattern',
  target: './lib/commands'
}
[14:34:34.096] DEBUG (sf:oclif:read-pjson): found oclif config in /usr/local/lib/sf/node_modules/@salesforce/plugin-deploy-retrieve/package.json
[14:34:34.096] DEBUG (sf:oclif:config:@salesforce/plugin-deploy-retrieve): command discovery options {
  globPatterns: [
    '**/*.+(js|cjs|mjs|ts|tsx|mts|cts)',
    '!**/*.+(d.ts|test.ts|test.js|spec.ts|spec.js|d.mts|d.cts)?(x)'
  ],
  strategy: 'pattern',
  target: './lib/commands'
}
[14:34:34.098] DEBUG (sf:oclif:config:@oclif/plugin-search): using manifest from /usr/local/lib/sf/node_modules/@oclif/plugin-search/oclif.manifest.json
[14:34:34.098] DEBUG (sf:oclif:config:@oclif/plugin-commands): using manifest from /usr/local/lib/sf/node_modules/@oclif/plugin-commands/oclif.manifest.json
[14:34:34.098] DEBUG (sf:oclif:config:@oclif/plugin-version): using manifest from /usr/local/lib/sf/node_modules/@oclif/plugin-version/oclif.manifest.json
[14:34:34.099] DEBUG (sf:oclif:config:@oclif/plugin-autocomplete): using manifest from /usr/local/lib/sf/node_modules/@oclif/plugin-autocomplete/oclif.manifest.json
[14:34:34.099] DEBUG (sf:oclif:config:@oclif/plugin-which): using manifest from /usr/local/lib/sf/node_modules/@oclif/plugin-which/oclif.manifest.json
[14:34:34.099] DEBUG (sf:oclif:config:@oclif/plugin-help): using manifest from /usr/local/lib/sf/node_modules/@oclif/plugin-help/oclif.manifest.json
[14:34:34.099] DEBUG (sf:oclif:config:@oclif/plugin-plugins): using manifest from /usr/local/lib/sf/node_modules/@oclif/plugin-plugins/oclif.manifest.json
[14:34:34.099] DEBUG (sf:oclif:config:@oclif/plugin-not-found): using manifest from /usr/local/lib/sf/node_modules/@oclif/plugin-not-found/oclif.manifest.json
[14:34:34.099] DEBUG (sf:oclif:config:@oclif/plugin-warn-if-update-available): using manifest from /usr/local/lib/sf/node_modules/@oclif/plugin-warn-if-update-available/oclif.manifest.json
[14:34:34.099] DEBUG (sf:oclif:config:@salesforce/plugin-auth): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-auth/oclif.manifest.json
[14:34:34.099] DEBUG (sf:oclif:config:@oclif/plugin-update): using manifest from /usr/local/lib/sf/node_modules/@oclif/plugin-update/oclif.manifest.json
[14:34:34.099] DEBUG (sf:oclif:config:@salesforce/plugin-marketplace): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-marketplace/oclif.manifest.json
[14:34:34.099] DEBUG (sf:oclif:config:@salesforce/plugin-info): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-info/oclif.manifest.json
[14:34:34.099] DEBUG (sf:oclif:config:@salesforce/plugin-data): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-data/oclif.manifest.json
[14:34:34.100] DEBUG (sf:oclif:config:@salesforce/plugin-apex): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-apex/oclif.manifest.json
[14:34:34.100] DEBUG (sf:oclif:config:@salesforce/plugin-schema): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-schema/oclif.manifest.json
[14:34:34.100] DEBUG (sf:oclif:config:@salesforce/plugin-settings): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-settings/oclif.manifest.json
[14:34:34.100] DEBUG (sf:oclif:config:@salesforce/plugin-org): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-org/oclif.manifest.json
[14:34:34.101] DEBUG (sf:oclif:config:@salesforce/plugin-sobject): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-sobject/oclif.manifest.json
[14:34:34.101] DEBUG (sf:oclif:config:@salesforce/plugin-packaging): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-packaging/oclif.manifest.json
[14:34:34.102] DEBUG (sf:oclif:config:@salesforce/plugin-source): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-source/oclif.manifest.json
[14:34:34.102] DEBUG (sf:oclif:config:@salesforce/plugin-telemetry): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-telemetry/oclif.manifest.json
[14:34:34.102] DEBUG (sf:oclif:config:@salesforce/plugin-limits): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-limits/oclif.manifest.json
[14:34:34.102] DEBUG (sf:oclif:config:@salesforce/plugin-trust): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-trust/oclif.manifest.json
[14:34:34.102] DEBUG (sf:oclif:config:@salesforce/plugin-user): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-user/oclif.manifest.json
[14:34:34.103] DEBUG (sf:oclif:config:@salesforce/plugin-templates): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-templates/oclif.manifest.json
[14:34:34.103] DEBUG (sf:oclif:config:@salesforce/plugin-deploy-retrieve): using manifest from /usr/local/lib/sf/node_modules/@salesforce/plugin-deploy-retrieve/oclif.manifest.json
[14:34:34.112] DEBUG (sf:oclif:config): config done
[14:34:34.112] DEBUG (sf:oclif:main): process.execPath: /usr/local/lib/sf/bin/node
[14:34:34.112] DEBUG (sf:oclif:main): process.execArgv: 
[14:34:34.113] DEBUG (sf:oclif:main): process.argv: [
  '/usr/local/lib/sf/bin/node',
  '/usr/local/lib/sf/bin/run',
  'version'
]
[14:34:34.113] DEBUG (sf:oclif:config): start init hook
[14:34:34.113] DEBUG (sf:oclif:config:ts-path): Skipping typescript path lookup for /usr/local/lib/sf/node_modules/@oclif/plugin-warn-if-update-available because NODE_ENV is NOT "test" or "development"
[14:34:34.113] DEBUG (sf:oclif:config:ts-path): Skipping typescript path lookup for /usr/local/lib/sf/node_modules/@oclif/plugin-update because NODE_ENV is NOT "test" or "development"
[14:34:34.113] DEBUG (sf:oclif:config:ts-path): Skipping typescript path lookup for /usr/local/lib/sf/node_modules/@salesforce/plugin-settings because NODE_ENV is NOT "test" or "development"
[14:34:34.119] DEBUG (sf:oclif:@salesforce/plugin-settings:hooks:init): start (import) /usr/local/lib/sf/node_modules/@salesforce/plugin-settings/lib/hooks/init/load_config_meta.js
[14:34:34.119] DEBUG (sf:oclif:config:ts-path): Skipping typescript path lookup for /usr/local/lib/sf/node_modules/@salesforce/plugin-deploy-retrieve because NODE_ENV is NOT "test" or "development"
[14:34:34.122] DEBUG (sf:oclif:@oclif/plugin-update:hooks:init): start (import) /usr/local/lib/sf/node_modules/@oclif/plugin-update/dist/hooks/init.js
[14:34:34.122] DEBUG (sf:oclif:@oclif/plugin-update:hooks:init): done
[14:34:34.122] DEBUG (sf:oclif:@oclif/plugin-warn-if-update-available:hooks:init): start (import) /usr/local/lib/sf/node_modules/@oclif/plugin-warn-if-update-available/lib/hooks/init/check-update.js
[14:34:34.140] DEBUG (sf:oclif:@salesforce/plugin-settings:hooks:init): done
[14:34:34.141] DEBUG (sf:oclif:@oclif/plugin-warn-if-update-available:hooks:init): done
[14:34:34.141] DEBUG (sf:oclif:config): init hook done
[14:34:34.142] DEBUG (sf:oclif:config): runCommand version [ [length]: 0 ]
[14:34:34.142] DEBUG (sf:oclif:config:ts-path): Skipping typescript path lookup for /usr/local/lib/sf/node_modules/@oclif/plugin-version because NODE_ENV is NOT "test" or "development"
[14:34:34.143] DEBUG (sf:oclif:config:@oclif/plugin-version): (import) /usr/local/lib/sf/node_modules/@oclif/plugin-version/lib/commands/version.js
[14:34:34.143] DEBUG (sf:oclif:config): start prerun hook
[14:34:34.143] DEBUG (sf:oclif:config:ts-path): Skipping typescript path lookup for /usr/local/lib/sf because NODE_ENV is NOT "test" or "development"
[14:34:34.143] DEBUG (sf:oclif:config:ts-path): Skipping typescript path lookup for /usr/local/lib/sf/node_modules/@salesforce/plugin-telemetry because NODE_ENV is NOT "test" or "development"
[14:34:34.144] DEBUG (sf:oclif:@salesforce/cli:hooks:prerun): start (import) /usr/local/lib/sf/dist/hooks/prerun.js
[14:34:34.144] DEBUG (sf:oclif:@salesforce/cli:hooks:prerun): done
[14:34:34.147] DEBUG (sf:oclif:@salesforce/plugin-telemetry:hooks:prerun): start (import) /usr/local/lib/sf/node_modules/@salesforce/plugin-telemetry/lib/hooks/telemetryPrerun.js
[14:34:34.147] DEBUG (sf:oclif:Config): Reading config file: /Users/username/.sf/config.json because hasRead is false
[14:34:34.147] DEBUG (sf:oclif:core:config:stateFromSfdxFileSync): Error reading state from sfdx config file at /Users/username/.sfdx/sfdx-config.json: ENOENT: no such file or directory, open '/Users/username/.sfdx/sfdx-config.json'
[14:34:34.147] DEBUG (sf:oclif:Config): Reading config file: /Users/username/Documents/Projects/sfdc-1/.sf/config.json because hasRead is false
@salesforce/cli/2.50.6 darwin-arm64 node-v20.15.0
[14:34:34.385] DEBUG (sf:oclif:@salesforce/plugin-telemetry:hooks:prerun): done
[14:34:34.385] DEBUG (sf:oclif:config): prerun hook done
[14:34:34.385] DEBUG (sf:oclif:sf:version): init version: @oclif/core@4.0.9 argv: [ [length]: 0 ]
[14:34:34.385] DEBUG (sf:oclif:config): start preparse hook
[14:34:34.385] DEBUG (sf:oclif:config:ts-path): Skipping typescript path lookup for /usr/local/lib/sf because NODE_ENV is NOT "test" or "development"
[14:34:34.386] DEBUG (sf:oclif:@salesforce/cli:hooks:preparse): start (import) /usr/local/lib/sf/dist/hooks/preparse.js
[14:34:34.386] DEBUG (sf:oclif:@salesforce/cli:hooks:preparse): done
[14:34:34.386] DEBUG (sf:oclif:config): preparse hook done
[14:34:34.387] DEBUG (sf:oclif:config): start postrun hook
[14:34:34.387] DEBUG (sf:oclif:config): postrun hook done
[14:34:34.388] DEBUG (sf:oclif:perf): Process Uptime: 456.1680ms
[14:34:34.388] DEBUG (sf:oclif:perf): Oclif Time: 274.3974ms
[14:34:34.388] DEBUG (sf:oclif:perf): Init Time: 29.2360ms
[14:34:34.388] DEBUG (sf:oclif:perf): Config Load Time: 60.1500ms
[14:34:34.388] DEBUG (sf:oclif:perf):   • Root Plugin Load Time: 11.3011ms
[14:34:34.388] DEBUG (sf:oclif:perf):   • Plugins Load Time: 38.8527ms
[14:34:34.388] DEBUG (sf:oclif:perf):   • Commands Load Time: 8.8774ms
[14:34:34.388] DEBUG (sf:oclif:perf): Core Plugin Load Time: 27.7975ms
[14:34:34.388] DEBUG (sf:oclif:perf): User Plugin Load Time: 1.1119ms
[14:34:34.388] DEBUG (sf:oclif:perf): Linked Plugin Load Time: 0.0000ms
[14:34:34.388] DEBUG (sf:oclif:perf): Plugin Load Times:
[14:34:34.388] DEBUG (sf:oclif:perf):   @oclif/plugin-autocomplete: 22.8632ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-deploy-retrieve: 22.5393ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @oclif/plugin-commands: 22.1360ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @oclif/plugin-help: 21.9147ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @oclif/plugin-not-found: 21.6884ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @oclif/plugin-plugins: 21.3493ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @oclif/plugin-update: 20.9177ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-limits: 20.6602ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @oclif/plugin-search: 20.5172ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-apex: 20.4184ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @oclif/plugin-version: 20.0853ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @oclif/plugin-warn-if-update-available: 19.9925ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @oclif/plugin-which: 19.6110ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-data: 19.3745ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-auth: 19.3224ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-packaging: 18.8175ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-info: 18.1697ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-org: 17.9793ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-templates: 17.6482ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-source: 17.4700ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-sobject: 17.1575ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-marketplace: 17.1520ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-telemetry: 16.9895ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-trust: 16.8487ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-schema: 16.7086ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-user: 16.6340ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @salesforce/plugin-settings: 16.5228ms
[14:34:34.388] DEBUG (sf:oclif:perf):   root: 11.3011ms
[14:34:34.388] DEBUG (sf:oclif:perf):   @oclif/core: 1.0048ms (no manifest!)
[14:34:34.388] DEBUG (sf:oclif:perf): Hook Run Times:
[14:34:34.388] DEBUG (sf:oclif:perf):   init:
[14:34:34.388] DEBUG (sf:oclif:perf):     total: 28.8190ms
[14:34:34.388] DEBUG (sf:oclif:perf):     @oclif/plugin-warn-if-update-available(./lib/hooks/init/check-update): 28.7015ms
[14:34:34.388] DEBUG (sf:oclif:perf):     @oclif/plugin-update(./dist/hooks/init.js): 8.9494ms
[14:34:34.388] DEBUG (sf:oclif:perf):     @salesforce/plugin-settings(./lib/hooks/init/load_config_meta): 27.5444ms
[14:34:34.388] DEBUG (sf:oclif:perf):   prerun:
[14:34:34.388] DEBUG (sf:oclif:perf):     total: 241.7680ms
[14:34:34.388] DEBUG (sf:oclif:perf):     @salesforce/cli(./dist/hooks/prerun): 1.1761ms
[14:34:34.388] DEBUG (sf:oclif:perf):     @salesforce/plugin-telemetry(./lib/hooks/telemetryPrerun.js): 241.5782ms
[14:34:34.388] DEBUG (sf:oclif:perf):   preparse:
[14:34:34.388] DEBUG (sf:oclif:perf):     total: 0.7032ms
[14:34:34.388] DEBUG (sf:oclif:perf):     @salesforce/cli(/dist/hooks/preparse): 0.6672ms
[14:34:34.388] DEBUG (sf:oclif:perf):   postrun:
[14:34:34.388] DEBUG (sf:oclif:perf):     total: 0.0278ms
[14:34:34.388] DEBUG (sf:oclif:perf): Command Load Time: 1.2915ms
[14:34:34.388] DEBUG (sf:oclif:perf): Command Run Time: 245.0381ms
```

**Comment by mdonnalley** [comment]
>It appears unsetting may have resolved it. Could you ELI5 on what had happened? Thanks!

[oclif](https://oclif.io/) (the framework that `sf` is built on) tries to load `devPlugins` if `NODE_ENV=development`. These plugins are only used for development purposes and aren't bundled with the CLI - so they'll exist in a local dev environment but not in a production environment. Since you had that env var set, oclif was trying to load those plugins but couldn't find them, which is what caused the `couldn't find package.json with name <plugin-name>` errors

Hopefully that helps!

**Comment by TNT-c** [comment]
> > It appears unsetting may have resolved it. Could you ELI5 on what had happened? Thanks!
> 
> [oclif](https://oclif.io/) (the framework that `sf` is built on) tries to load `devPlugins` if `NODE_ENV=development`. These plugins are only used for development purposes and aren't bundled with the CLI - so they'll exist in a local dev environment but not in a production environment. Since you had that env var set, oclif was trying to load those plugins but couldn't find them, which is what caused the `couldn't find package.json with name <plugin-name>` errors
> 
> Hopefully that helps!

Thanks king
